youtube_dl/downloader/__init__.py

```diff
@@ -30,6 +31,9 @@ def get_suitable_downloader(info_dict, params={}):
     protocol = determine_protocol(info_dict)
     info_dict['protocol'] = protocol
 
+    if (info_dict.get('start_time') or info_dict.get('end_time')) and FFmpegFD.supports(info_dict):
```

Comment:
i tested dash video+audio the video and audio are out of sync.
but it works with this command(at least for the tests that i run on youtube):
`ffmpeg -ss start -t end video_url -ss start -t end audio_url -c copy out.ext`